File: Persistence/DB/response/TeamIndivMatch.cs
namespace DB.SportHive.Domain
{
    public class TeamIndivMatch
    {
        public string TypeSport { get; set; } = null!;
        public List<TeamIndivMatchRes> Matches { get; set; } = null!;
    }
    public class TeamIndivMatchRes
    {
        public string firstTeamScore { get; set; } = null!;
        public string secondTeamScore { get; set;} = null!;
        public string NameEntity1 { get; set; } = null!;
        public string photoFirstEntity { get; set; } = null!;
        public string NameEntity2 { get; set; } = null!;
        public string photoSecondEntity { get; set; } = null!;
        public string statusMatch { get; set; } = null!;
    }
}

File: Frontend/src/logic/TeamIndivMatch.ts
export interface TeamIndivMatch {
    typeSport: string;
    matches: TeamIndivMatchRes[];
}

export interface TeamIndivMatchRes {
    firstTeamScore: string;
    secondTeamScore: string;
    nameEntity1: string;
    photoFirstEntity: string;
    nameEntity2: string;
    photoSecondEntity: string;
    statusMatch: string; // "Win", "Loss", "Draw" або подібне
}
